usr.sbin/adduser/adduser.sh

```diff
@@ -613,6 +654,84 @@ get_password() {
 	fi
 }
 
+# get_zfs_encryption
+#	Read ZFS options for the user's home dataset.
+#
+get_zfs_encryption() {
+	_input=
+	_prompt="Enable ZFS encryption? (yes/no) [no]: "
+	while : ; do
+		echo -n "$_prompt"
+		read _input
+
+		[ -z "$_input" ] && _input=Zencrypt
```

Comment:
The prompt describes "no" as the default, but this seems to be trying to use the current value of `Zencrypt` as the default? (But it should have been `$Zencrypt`... this should probably be `no` to match the prompt, though)

Reply:
That's a bug, fixed.